Source/Core/VideoCommon/Statistics.cpp

```diff
@@ -336,12 +336,12 @@ void Statistics::DisplayScissor()
 
     // Visualization of where things are updated on screen with this specific scissor
     ImGui::TableNextColumn();
-    float scale = ImGui::GetTextLineHeight() / EFB_HEIGHT;
+    float scale2 = ImGui::GetTextLineHeight() / EFB_HEIGHT;
```

Comment:
consider `scaled_height` instead of `scale2`, since it makes it harder to introduce non-obvious bugs in the event `scale` is accidentally used